analytics/tests/test_counts.py

```diff
@@ -11,61 +10,63 @@
     zerver_count_message_by_huddle
 from analytics.models import UserCount, RealmCount, StreamCount, InstallationCount, Stream, Recipient
 
-from zerver.lib.test_helpers import make_client, get_stream
-from zerver.models import Realm, UserProfile, Message, get_user_profile_by_email
+from zerver.models import Realm, UserProfile, Message, get_user_profile_by_email, get_client
 
 from typing import Any
 from six import text_type
 
-def do_update_past_hour(stat):
-    # type: (CountStat) -> Any
-    return process_count_stat(stat, range_start=timezone.now() - timedelta(seconds=3600),
-                              range_end=timezone.now())
+class AnalyticsTestCase(TestCase):
+    MINUTE = timedelta(seconds = 60)
+    HOUR = MINUTE * 60
+    DAY = HOUR * 24
+    TIME_ZERO = datetime(2042, 3, 14).replace(tzinfo=timezone.utc)
+    TIME_LAST_HOUR = TIME_ZERO - HOUR
 
-class TestDataCollectors(TestCase):
+    def process_last_hour(self, stat):
+        # type: (CountStat) -> None
+        process_count_stat(stat, self.TIME_ZERO, self.TIME_ZERO)
```

Comment:
These eventually become the first and last arguments of timeinterval_range, which is an inclusive range. Can add a comment.